src/api/public/apidocs-new/paths/source_project_name_package_name.yaml

```diff
@@ -1,3 +1,352 @@
+                07070100000000000081a40000000000000000000000015b595e440000006a000000000000000000000000000000000000000c00000000_multibuild<multibuild>
+                  <flavor>obs-bundled-gems</flavor>
+                  <flavor>obs-api-testsuite-rspec</flavor>
+                </multibuild>
+                07070100000000000081a40000000000000000000000016229cffc000002ec000000000000000000000000000000000000000900000000_service<?xml version="1.0"?>
+                <services>
+                  <service name="obs_scm">
+                    <param name="versionformat">2.11~alpha.%ci.%h</param>
+                    <param name="url">https://github.com/openSUSE/open-build-service.git</param>
+                    <param name="scm">git</param>
+                    <param name="revision">master</param>
+                    <param name="extract">dist/obs-api-testsuite-rspec.spec</param>
+                    <param name="extract">dist/obs-bundled-gems.spec</param>
+                    <param name="extract">dist/obs-server.spec</param>
+                    <param name="extract">dist/obs-rpmlintrc</param>
+                  </service>
+                  <service name="set_version"/>
+                  <service name="tar" mode="buildtime"/>
+                  <service name="recompress" mode="buildtime">
+                    <param name="compression">xz</param>
+                    <param name="file">*.tar</param>
+                  </service>
+                </services>
+                07070100000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000b00000000TRAILER!!!
+    '400':
+      description: |
+        Bad Request.
+
+        XML Schema used for body validation: [status.xsd](../schema/status.xsd)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Not Boolean:
+              description: Passing a value different than `0` or `1` to `meta`, for example.
+              value:
+                code: 400
+                summary: not boolean
+            With deleted=1 and Package exists:
+              description: Passing a value of `1` to `deleted`, and the package exists.
+              value:
+                code: package_exists
+                summary: the package is not deleted
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      description: Not Found.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Unknown Project:
+              value:
+                code: unknown_project
+                summary: "Project not found: home:some_project"
+            Unknown Package:
+              value:
+                code: unknown_package
+                summary: "Package not found: home:some_project/some_package"
```

Comment:
You could use a reference here, right?
```
  '404':
      $ref: '../components/responses/unknown_project_or_package.yaml'
```

Reply:
Also applied, thanks! Please take a look again.